lms/djangoapps/ccx/tests/test_field_override_performance.py

```diff
@@ -46,6 +46,7 @@ class FieldOverridePerformanceTestCase(ProceduralCourseTestMixin,
     providers.
     """
     __test__ = False
+    multi_db = True
```

Comment:
multi_db is code Django, but unusual unless you've used it.  I've added a canned comment to the 10 places I do it to help future explorers.